server/public/partials/statblock/statBlock.html

```diff
@@ -152,13 +152,12 @@ <h4>Languages</h4>
           <span ng-repeat="language in creature.stats.languages"><span ng-if="$index>0">, </span>{{language}}</span>
         </p>
       </div>
-      <div class="property-line">
-        <h4 style="opacity: 0;">Challenge</h4>
-        <div class="property-line" style="float: left; margin-left: -3.5%">
+      <div class="property-line" style="margin-bottom: 1.33em">
+        <div class="property-line" style="float: left; margin-left: -1em">
           <h4>Challenge</h4>
           <p>{{creature.stats.challengeRatingStr}} ({{creature.stats.experiencePoints}} XP)</p>
         </div>
-        <div class="property-line" style="float: right; margin-right: 15%">
+        <div class="property-line" style="display:inline-block;margin-left: 42px">
```

Comment:
This mirrors the same design used by dndbeyond to place the proficiency next to the xp. Looks good on my PC and mobile device

Reply:
The proficiency bonus is not pulled all the way to the right, it's just 42px further than the Challenge rating. This doesn't look quite right:
![Screenshot 2021-12-11 163853](https://user-images.githubusercontent.com/7504022/145696431-52cbacb9-a1e7-4887-a654-9d35b1bb5381.png)

Instead, I would recommend trying something like:
```html
<div class="property-line" style="padding-left: 0; padding-right: 12px;">
        <div class="property-line" style="display: inline-block; text-indent: 0px;">
          <h4>Challenge</h4>
          <p>{{creature.stats.challengeRatingStr}} ({{creature.stats.experiencePoints}} XP)</p>
        </div>
        <div class="property-line" style="display: inline-block; float: right;">
          <h4>Proficiency Bonus</h4>
          <p>+{{creature.stats.proficiencyBonus}}</p>
        </div>
      </div>
```

I would also recommend making those into CSS classes. In theory, this format of left/right blocks should be reusable.

Reply:
I see what you are saying. I was trying to mimic what dndbeyond did, but in truth there wasn't a good reason not to build something that looks better, like your example.  I'll create CSS classes to make this reusable and try to implement this solution